.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Version Release
```

Comment:
Ah I see what you are doing. Thanks for writing this, it's really cool! I hope it wasn't taking too much of your time. We really appreciate community contributions ❤️ and I can see you put a lot of effort into this for the community. The way releases work in github is not by having a specific release pipeline (like you would in other build services), but instead by pointing at a specific artifact and giving it a release label. So it still makes sense to have a release pipeline in situations when you want to make a specially tested build. But this project doesn't have that. So I think the basic CICD should be sufficient.

Reply:
Thank **_you_** guys for making such an awesome database, your work makes an impact on the community and we have to give back ❤️ . 

Regarding the release mechanism, I understand this might not be the behaviour you want, but I was trying to imitate the behavior of the `surrealdb.js` repo, see: https://github.com/surrealdb/surrealdb.js/blob/main/.github/workflows/publish.yml Anyway, I will delete this workflow from this PR so that this PR can be merged.

Thanks